src/main/resources/locale_de_DE.properties

```diff
@@ -217,36 +217,40 @@ command.repsettings.info.message.title=Reputationseinstellungen
-command.roles.sub.remove=Entfernt eine Reputationsrolle.
-command.roles.sub.remove.arg.role=Die Rolle, die nicht mehr für die Reputation verwendet werden soll.
-command.roles.sub.remove.notARepRole=Diese Rolle ist keine Reputationsrolle.
-command.roles.sub.remove.removed=Die Rolle %ROLE% wurde entfernt.
-command.roles.sub.removeDonor=Entfernt eine Spender-Rolle.
-command.roles.sub.removeDonor.arg.role=Die Spenderrolle, die entfernt werden soll.
-command.roles.sub.removeDonor.remove=Spender-Rolle %ROLE% entfernt.
-command.roles.sub.removeReceiver=Entfernt eine Empfänger-Rolle.
-command.roles.sub.removeReceiver.arg.role=Die Empfängerrolle, die entfernt werden soll.
-command.roles.sub.removeReceiver.remove=Empfänger-Rolle %ROLE% entfernt.
-command.roles.sub.stackRoles=Erlaubt das Stapeln von Rollen
-command.roles.sub.stackRoles.arg.stack=Die Rolle, die gestapelt werden soll, wird auf "true" gesetzt.
-command.roles.sub.stackRoles.notStacked=Nutzer werden nur die Rolle mit der höchsten Reputation haben.
-command.roles.sub.stackRoles.stacked=Rollen werden gestapelt. Nutzer haben alle Rollen, die weniger Reputation benötigen, als sie haben.
+command.roles.info.message.donorroles=Spender-Rollen
+command.roles.info.message.receiverroles=Empfänger-Rollen
+command.roles.info.message.reputationrole=Reputation-Rollen
+command.roles.info.message.roleinfo=Rollen-Infos
+command.roles.list.description=Listet die Reputationsrollen auf.
+command.roles.refresh.description=Aktualisiert alle Nutzer-Rollen
+command.roles.refresh.message.finished=Aktualisieren der Rollen ist abgeschlossen.
+command.roles.refresh.message.running=Es ist bereits eine Rollen-Aktualisierung in Arbeit.
+command.roles.refresh.message.started=Rollen-Aktualisierung gestartet.
+command.roles.remove.description=Entfernt eine Reputationsrolle.
+command.roles.remove.message.noreprole=Diese Rolle ist keine Reputationsrolle.
+command.roles.remove.message.removed=Die Rolle %ROLE% wurde entfernt.
+command.roles.remove.role.description=Die Rolle, die nicht mehr für die Reputation verwendet werden soll.
+command.roles.removedonor.description=Entfernt eine Spender-Rolle.
+command.roles.removedonor.role.description=Die Spenderrolle, die entfernt werden soll.
+command.roles.removereceiver.description=Entfernt eine Empfänger-Rolle.
+command.roles.removereceiver.message.remove=Empfänger-Rolle %ROLE% entfernt.
+command.roles.removereceiver.role.description=Die Empfängerrolle, die entfernt werden soll.
+command.roles.removeronor.message.remove=Spender-Rolle %ROLE% entfernt.
+command.roles.stackroles.description=Erlaubt das Stapeln von Rollen
+command.roles.stackroles.message.notStacked=Nutzer werden nur die Rolle mit der höchsten Reputation haben.
+command.roles.stackroles.message.stacked=Rollen werden gestapelt. Nutzer haben alle Rollen, die weniger Reputation benötigen, als sie haben.
+command.roles.stackroles.stack.description=Die Rolle, die gestapelt werden soll, wird auf "true" gesetzt.
```

Comment:
Wow, sometimes we just have total garbage in our german translation :joy:

Reply:
Thats why I love it!